common/src/main/scala/org/apache/celeborn/common/RssConf.scala

```diff
@@ -1412,25 +1514,44 @@ object RssConf extends Logging {
       .toSequence
       .createWithDefaultString("readwrite,diskusage")
 
-  def deviceMonitorEnabled(conf: RssConf): Boolean = conf.get(WORKER_DEVICE_MONITOR_ENABLED)
-
-  def deviceMonitorCheckList(conf: RssConf): Seq[String] = conf.get(WORKER_DEVICE_MONITOR_CHECKLIST)
+  val WORKER_DISK_CHECK_INTERVAL: ConfigEntry[Long] =
+    buildConf("celeborn.worker.diskmonitor.check.interval")
```

Comment:
I used the namespace `celeborn.worker.deviceMonitor.*` in the previous PR, what's your suggestion here?

Reply:
if we are going to monitor other things in the future, like cpu load, I would suggest using `celeborn.worker.monitor.disk.*` here

Reply:
This is a typo. And I'll change it.

Reply:
cc @waitinfuture as well

Reply:
> 

As we discussed earlier, I prefer the second one 
```
celeborn.worker.monitor.disk.*
```